client/src/components/nav_comp/DemoLoginButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { login } from '../../store/auth';
+import { Redirect } from 'react-router-dom';
+import '../../style/loginmodal.css'
+
+function LoginButton(){
+    const username = "yusuke@example.com"
+    const password = "password"
+    const currentUserId = useSelector(state => state.auth.id);
+    const dispatch = useDispatch();
+    
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        dispatch(login(username, password))
+    }
+
+    if(currentUserId) return <Redirect to="/" />;
+
+    return(
+        <div className="login" onClick={handleSubmit}>
+            <button id="demologin" >Demo Sign in</button>
```

Comment:
I would also suggest setting the `onClick` listener for the `<button>`, instead of the `<div>`!

```suggestion
        <div className="login">
            <button id="demologin" onClick={handleSubmit}>Demo Sign in</button>
```